src/compiler/crystal/macros/methods.cr

```diff
@@ -145,10 +145,18 @@ module Crystal
     def interpret_flag?(node)
       if node.args.size == 1
         node.args[0].accept self
-        flag = @last.to_macro_id
-        @last = BoolLiteral.new(@program.has_flag?(flag))
+        flag_name = @last.to_macro_id
+        flags = case node.name
```

Comment:
Minor improvement: `flags` could be passed as an argument to this method, so we don't need to compare the node name again here. Instead, make it two separate branches in the already existing `case`.

Reply:
No, just look at the code around it, it's a very strong pattern